docs/concepts/flows.md

```diff
@@ -991,6 +991,15 @@ While the cancellation process is robust, there are a few issues than can occur:
 - If the `infrastructre_pid` is missing from the flow run will be marked as cancelled but cancellation cannot be enforced.
 - If the agent runs into an unexpected error during cancellation the flow run may or may not be cancelled depending on where the error occurred. The agent will try again to cancel the flow run. Another agent may attempt cancellation.
 
+!!! tip "Enhanced cancellation"
+    We are working on improving cases where cancellation can fail. You can try the improved cancellation experience by enabling the 
+ 'PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION' setting on your worker or agents:
```

Comment:
```suggestion
!!! tip "Enhanced cancellation"
    We are working on improving cases where cancellation can fail. You can try the improved cancellation experience by enabling the 'PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION' setting on your worker or agents:
```

Reply:
<img width="733" alt="Screenshot 2023-10-12 at 3 18 19 PM" src="https://github.com/PrefectHQ/prefect/assets/42048900/83eecec2-114e-41f3-9b97-cc3b2999e6df">

preview is looking funky